### PR TITLE
Use "main" branch of islandora-sandbox for Install Profile ("demo").

### PR DESCRIPTION
**Pull request by rosiel**
A long time ago, the `install-profile` branch of the Islandora-Sandbox was the thing to install if you wanted to get the Install Profile. The PR was open for a long time and "use the install-profile branch" became the best practice. At that point it must have been merged into the playbook.

[ISLE-DC now uses the  main branch](https://github.com/Islandora-Devops/isle-dc/blob/development/Makefile#L355), so this updates the playbook to do the same.

@alxp @whikloj @g7morris

### CODE REVIEW

**Review by seth-shaw-asu — DISMISSED** [review]
Works for me.

**Comment by seth-shaw-asu** [comment]
Works for me. Any objections from the @Islandora-Devops/committers before I merge?

**Comment by seth-shaw-asu** [comment]
Actually, I may have spoken too soon...

**Comment by seth-shaw-asu** [comment]
Yup, I spoke too soon. Ansible crashes on the migrations:
```
TASK [Run migrations] **********************************************************
Wednesday 01 March 2023  16:29:55 -0800 (0:00:01.149)       0:11:07.848 ******* 
fatal: [default]: FAILED! => {"changed": true, "cmd": ["/var/www/html/drupal/vendor/bin/drush", "--root", "/var/www/html/drupal/web", "-y", "-l", "localhost:8000", "--userid=1", "mim", "islandora_tags,islandora_defaults_tags"], "delta": "0:00:07.348268", "end": "2023-03-02 00:30:02.910067", "msg": "non-zero return code", "rc": 255, "start": "2023-03-02 00:29:55.561799", "stderr": " [notice] Now acting as user ID 1\n\n  1/13 [▓▓░░░░░░░░░░░░░░░░░░░░░░░░░░]   7%\n  2/13 [▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░]  15%\n  3/13 [▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░]  23%\n  4/13 [▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░]  30%\n  6/13 [▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░]  46%\n  7/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░]  53%\n  8/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░]  61%\n 10/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░]  76%\n 11/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░]  84%\n 12/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░]  92%\n 13/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100% [notice] Processed 13 items (13 created, 0 updated, 0 failed, 0 ignored) in 2.8 seconds (281.7/min) - done with 'islandora_defaults_tags'\n\n  1/18 [▓░░░░░░░░░░░░░░░░░░░░░░░░░░░]   5%\n  2/18 [▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░░]  11%\n  4/18 [▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░]  22%\n  6/18 [▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░]  33%\n  8/18 [▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░]  44%\n  9/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░]  50%\n 11/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░]  61%\n 13/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░]  72%\n 15/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░]  83%\n 17/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░]  94%\n 18/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100% [notice] Processed 18 items (18 created, 0 updated, 0 failed, 0 ignored) in 3.2 seconds (333.1/min) - done with 'islandora_tags'\n [notice] Switching back from user 1.\n [warning] Undefined array key \"specific_languages\" SearchApiSolrBackend.php:1195\nPHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /var/www/html/drupal/web/modules/contrib/search_api_solr/src/Plugin/search_api/backend/SearchApiSolrBackend.php:1195\nStack trace:\n#0 /var/www/html/drupal/web/modules/contrib/search_api_solr/src/Plugin/search_api/backend/SearchApiSolrBackend.php(1195): array_filter(NULL)\n#1 /var/www/html/drupal/web/modules/contrib/search_api_solr/src/Plugin/search_api/backend/SearchApiSolrBackend.php(1118): Drupal\\search_api_solr\\Plugin\\search_api\\backend\\SearchApiSolrBackend->getDocuments(Object(Drupal\\search_api\\Entity\\Index), Array, Object(Solarium\\QueryType\\Update\\Query\\Query))\n#2 /var/www/html/drupal/web/modules/contrib/search_api/src/Entity/Server.php(350): Drupal\\search_api_solr\\Plugin\\search_api\\backend\\SearchApiSolrBackend->indexItems(Object(Drupal\\search_api\\Entity\\Index), Array)\n#3 /var/www/html/drupal/web/modules/contrib/search_api/src/Entity/Index.php(994): Drupal\\search_api\\Entity\\Server->indexItems(Object(Drupal\\search_api\\Entity\\Index), Array)\n#4 /var/www/html/drupal/web/modules/contrib/search_api/src/Utility/PostRequestIndexing.php(92): Drupal\\search_api\\Entity\\Index->indexSpecificItems(Array)\n#5 /var/www/html/drupal/web/core/lib/Drupal/Core/EventSubscriber/KernelDestructionSubscriber.php(51): Drupal\\search_api\\Utility\\PostRequestIndexing->destruct()\n#6 [internal function]: Drupal\\Core\\EventSubscriber\\KernelDestructionSubscriber->onKernelTerminate(Object(Symfony\\Component\\HttpKernel\\Event\\TerminateEvent), 'kernel.terminat...', Object(Drupal\\Component\\EventDispatcher\\ContainerAwareEventDispatcher))\n#7 /var/www/html/drupal/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php(142): call_user_func(Array, Object(Symfony\\Component\\HttpKernel\\Event\\TerminateEvent), 'kernel.terminat...', Object(Drupal\\Component\\EventDispatcher\\ContainerAwareEventDispatcher))\n#8 /var/www/html/drupal/vendor/symfony/http-kernel/HttpKernel.php(103): Drupal\\Component\\EventDispatcher\\ContainerAwareEventDispatcher->dispatch(Object(Symfony\\Component\\HttpKernel\\Event\\TerminateEvent), 'kernel.terminat...')\n#9 /var/www/html/drupal/vendor/stack/builder/src/Stack/StackedHttpKernel.php(32): Symfony\\Component\\HttpKernel\\HttpKernel->terminate(Object(Symfony\\Component\\HttpFoundation\\Request), Object(Symfony\\Component\\HttpFoundation\\Response))\n#10 /var/www/html/drupal/web/core/lib/Drupal/Core/DrupalKernel.php(687): Stack\\StackedHttpKernel->terminate(Object(Symfony\\Component\\HttpFoundation\\Request), Object(Symfony\\Component\\HttpFoundation\\Response))\n#11 /var/www/html/drupal/vendor/drush/drush/src/Boot/DrupalBoot8.php(298): Drupal\\Core\\DrupalKernel->terminate(Object(Symfony\\Component\\HttpFoundation\\Request), Object(Symfony\\Component\\HttpFoundation\\Response))\n#12 [internal function]: Drush\\Boot\\DrupalBoot8->terminate()\n#13 {main}\n  thrown in /var/www/html/drupal/web/modules/contrib/search_api_solr/src/Plugin/search_api/backend/SearchApiSolrBackend.php on line 1195", "stderr_lines": [" [notice] Now acting as user ID 1", "", "  1/13 [▓▓░░░░░░░░░░░░░░░░░░░░░░░░░░]   7%", "  2/13 [▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░]  15%", "  3/13 [▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░]  23%", "  4/13 [▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░]  30%", "  6/13 [▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░]  46%", "  7/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░]  53%", "  8/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░]  61%", " 10/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░]  76%", " 11/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░]  84%", " 12/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░]  92%", " 13/13 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100% [notice] Processed 13 items (13 created, 0 updated, 0 failed, 0 ignored) in 2.8 seconds (281.7/min) - done with 'islandora_defaults_tags'", "", "  1/18 [▓░░░░░░░░░░░░░░░░░░░░░░░░░░░]   5%", "  2/18 [▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░░]  11%", "  4/18 [▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░]  22%", "  6/18 [▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░]  33%", "  8/18 [▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░]  44%", "  9/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░]  50%", " 11/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░]  61%", " 13/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░]  72%", " 15/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░]  83%", " 17/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░]  94%", " 18/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100% [notice] Processed 18 items (18 created, 0 updated, 0 failed, 0 ignored) in 3.2 seconds (333.1/min) - done with 'islandora_tags'", " [notice] Switching back from user 1.", " [warning] Undefined array key \"specific_languages\" SearchApiSolrBackend.php:1195", "PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /var/www/html/drupal/web/modules/contrib/search_api_solr/src/Plugin/search_api/backend/SearchApiSolrBackend.php:1195", "Stack trace:", "#0 /var/www/html/drupal/web/modules/contrib/search_api_solr/src/Plugin/search_api/backend/SearchApiSolrBackend.php(1195): array_filter(NULL)", "#1 /var/www/html/drupal/web/modules/contrib/search_api_solr/src/Plugin/search_api/backend/SearchApiSolrBackend.php(1118): Drupal\\search_api_solr\\Plugin\\search_api\\backend\\SearchApiSolrBackend->getDocuments(Object(Drupal\\search_api\\Entity\\Index), Array, Object(Solarium\\QueryType\\Update\\Query\\Query))", "#2 /var/www/html/drupal/web/modules/contrib/search_api/src/Entity/Server.php(350): Drupal\\search_api_solr\\Plugin\\search_api\\backend\\SearchApiSolrBackend->indexItems(Object(Drupal\\search_api\\Entity\\Index), Array)", "#3 /var/www/html/drupal/web/modules/contrib/search_api/src/Entity/Index.php(994): Drupal\\search_api\\Entity\\Server->indexItems(Object(Drupal\\search_api\\Entity\\Index), Array)", "#4 /var/www/html/drupal/web/modules/contrib/search_api/src/Utility/PostRequestIndexing.php(92): Drupal\\search_api\\Entity\\Index->indexSpecificItems(Array)", "#5 /var/www/html/drupal/web/core/lib/Drupal/Core/EventSubscriber/KernelDestructionSubscriber.php(51): Drupal\\search_api\\Utility\\PostRequestIndexing->destruct()", "#6 [internal function]: Drupal\\Core\\EventSubscriber\\KernelDestructionSubscriber->onKernelTerminate(Object(Symfony\\Component\\HttpKernel\\Event\\TerminateEvent), 'kernel.terminat...', Object(Drupal\\Component\\EventDispatcher\\ContainerAwareEventDispatcher))", "#7 /var/www/html/drupal/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php(142): call_user_func(Array, Object(Symfony\\Component\\HttpKernel\\Event\\TerminateEvent), 'kernel.terminat...', Object(Drupal\\Component\\EventDispatcher\\ContainerAwareEventDispatcher))", "#8 /var/www/html/drupal/vendor/symfony/http-kernel/HttpKernel.php(103): Drupal\\Component\\EventDispatcher\\ContainerAwareEventDispatcher->dispatch(Object(Symfony\\Component\\HttpKernel\\Event\\TerminateEvent), 'kernel.terminat...')", "#9 /var/www/html/drupal/vendor/stack/builder/src/Stack/StackedHttpKernel.php(32): Symfony\\Component\\HttpKernel\\HttpKernel->terminate(Object(Symfony\\Component\\HttpFoundation\\Request), Object(Symfony\\Component\\HttpFoundation\\Response))", "#10 /var/www/html/drupal/web/core/lib/Drupal/Core/DrupalKernel.php(687): Stack\\StackedHttpKernel->terminate(Object(Symfony\\Component\\HttpFoundation\\Request), Object(Symfony\\Component\\HttpFoundation\\Response))", "#11 /var/www/html/drupal/vendor/drush/drush/src/Boot/DrupalBoot8.php(298): Drupal\\Core\\DrupalKernel->terminate(Object(Symfony\\Component\\HttpFoundation\\Request), Object(Symfony\\Component\\HttpFoundation\\Response))", "#12 [internal function]: Drush\\Boot\\DrupalBoot8->terminate()", "#13 {main}", "  thrown in /var/www/html/drupal/web/modules/contrib/search_api_solr/src/Plugin/search_api/backend/SearchApiSolrBackend.php on line 1195"], "stdout": "", "stdout_lines": []}

PLAY RECAP *********************************************************************
default                    : ok=223  changed=120  unreachable=0    failed=1    skipped=319  rescued=0    ignored=1   

Wednesday 01 March 2023  16:30:02 -0800 (0:00:07.684)       0:11:15.533 ******* 
=============================================================================== 
geerlingguy.drupal : Install Drupal with drush. ----------------------- 119.50s
Islandora-Devops.crayfish : Build crayfish code including dependencies - 108.95s
Islandora-Devops.crayfish : Install requisite packages ----------------- 99.11s
geerlingguy.drupal : Run composer install if composer.json is present. -- 44.88s
Islandora-Devops.drupal-openseadragon : Install Openseadragon with composer -- 33.36s
Islandora-Devops.fits : Wait for FITS to finish coming back up --------- 24.59s
webserver-app : Import features ---------------------------------------- 18.90s
Islandora-Devops.fcrepo : Install python dependencies ------------------ 17.42s
Islandora-Devops.fits : Run Composer On Fits Microservice -------------- 15.88s
Islandora-Devops.fits : Downloading FITS executables ------------------- 12.16s
Restart SOLR to make config take effect. -------------------------------- 9.79s
Run migrations ---------------------------------------------------------- 7.68s
geerlingguy.drupal : Check out Drupal to the docroot. ------------------- 6.38s
Islandora-Devops.fcrepo : Download fcrepo war (post 5.0.0) -------------- 5.71s
gather facts ------------------------------------------------------------ 5.43s
webserver-app : Uninstall core search module ---------------------------- 4.96s
Islandora-Devops.fits : Unzip FITS -------------------------------------- 4.83s
Copy solr config files -------------------------------------------------- 4.39s
Gathering Facts --------------------------------------------------------- 4.20s
Gathering Facts --------------------------------------------------------- 4.06s
Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

**Comment by rosiel** [comment]
I'm also getting this. From the looks of the error message, it's not the migrations, it's indexing the content that fails, because the solr index config is missing a chunk of configuration that's required by search_api_solr, namely the `third_party_settings/search_api_solr/multilingual/specific_languages` key. 

I am guessing that ISLE-DC is getting around this by running a script called `create_solr_core_with_default_config` 

If this script is expected for the install profile to work, then it needs to be in the install profile's documentation. Or, better, fix the config in the config/install folder so that it works.

**Pull request closed without merging**
